extension/chrome/dev/ci_pgp_host_page.ts

```diff
@@ -0,0 +1,22 @@
+/* ©️ 2016 - present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com */
+
+'use strict';
+
+import { BrowserMsg } from '../../js/common/browser/browser-msg.js';
+import { Catch } from '../../js/common/platform/catch.js';
+import { PgpMsg } from '../../js/common/core/pgp-msg.js';
+import { Env } from '../../js/common/browser/env.js';
+
+/* eslint-disable max-len */
+
+Catch.try(async () => {
+  const tabId = await BrowserMsg.requiredTabId();
+
+  BrowserMsg.addListener('pgpMsgDiagnosePubkeys', PgpMsg.diagnosePubkeys);
+  BrowserMsg.listen(tabId);
+
+  let src = Env.getBaseUrl();
+  src += `/chrome/elements/pgp_block.htm?${location.search}`;
+  src += `&parentTabId=${encodeURIComponent(tabId)}`;
+  $('body').append(`<iframe src="${src}" frameborder="0"></iframe>`); // xss-direct
```

Comment:
While here it doesn't really matter, since it's just for dev, the correct way to fix it here would be `<iframe src="${Xss.escape(src)}" frameborder="0"></iframe>` with `// xss-escaped`